### ISSUE TITLE
Tabs Component Keyboard Navigation

### ISSUE THREAD

**Issue by pierrerocket** [twbs/bootstrap#34814]
Hello.  Our Accessibility auditor has brought up the issue that the tab behavior in bootstrap doesn't meet their expectations.

The expected behavior:
Tab (key) - once to navigate into the group of tabs and once to navigate out of the group of tabs
↑/↓ or ←/→ - choose and activate previous/next tab.

What is happening:
Tab (key) moves into the group, and tab moves between the tabs, and the arrow keys don't function at all.

https://webaim.org/techniques/keyboard/

**Comment by GeoSot** [comment]
Hey, I think it may be closed by #33079. Please make a check there and confirm
